utils/world.ts

```diff
@@ -60,6 +61,7 @@ export default class World {
   private minSpawnDistance = 10;
   private maxMoveDistance = 2;
   private newBotRadius = 5;
+  private startTime = new Date().getTime.toString();
```

Comment:
Опечатка: забыл вызвать `getTime()`:
```suggestion
  private startTime = new Date().getTime().toString();
```

Но, как я писал выше, лучше дату хранить в формате `Date`, так мы и храним её эффективно и всем сразу очевидно, что это дата.

```suggestion
  private startTime = new Date();
```

А, если нам надо её преобразовать в другой формат для записи в базу данных, это должен делать метод, записывающий в БД. Так сразу будет понятно зачем вообще мы производим эти преобразования.